recipes/libwebsockets/all/conanfile.py

```diff
@@ -422,6 +422,7 @@ def generate(self):
         tc.variables["LWS_WITH_SYS_SMD"] = self.settings.os != "Windows"
         tc.variables["DISABLE_WERROR"] = True
 
+        tc.cache_variables["CMAKE_POLICY_VERSION_MINIMUM"] = "3.5" # CMake 4 support
```

Comment:
it's interesting that this was fixed in master upstream ~2 years ago
https://github.com/warmcat/libwebsockets/blob/main/CMakeLists.txt

but none of the recent releases have it